CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+### Date:		2019-May-01
+### Release:	v2019050101
+
+#### :wrench: Fixes and enhancements
+
+#### Migration tool modal improvements
+
+We’ve made sure that the modal displays when using our migration tool. If you migration is successful, then a green model box will display to confirm this.
```

Comment:
Pedantic, I know... but sorry.
We have made sure that the modal displays... (what modal?)
Also "then a green model" should be "then a green modal"